frontend/src/scenes/pipeline/hogfunctions/logs/HogFunctionLogs.tsx

```diff
@@ -10,30 +11,87 @@ import { hogFunctionLogsLogic } from './hogFunctionLogsLogic'
 import { LogsViewer } from './LogsViewer'
 import { GroupedLogEntry, LogsViewerLogicProps } from './logsViewerLogic'
 
-const eventIdMatchers = [/Event: ([A-Za-z0-9-]+)/, /\/events\/([A-Za-z0-9-]+)\//]
-
 export function HogFunctionLogs(props: { hogFunctionId: string }): JSX.Element {
-    return (
-        <LogsViewer
-            sourceType="hog_function"
-            sourceId={props.hogFunctionId}
-            renderColumns={(columns) => {
-                // Add in custom columns for handling retries
-                const newColumns: LemonTableColumns<GroupedLogEntry> = [
-                    {
-                        title: 'Status',
-                        key: 'status',
-                        width: 0,
-                        render: (_, record) => (
-                            <HogFunctionLogsStatus record={record} hogFunctionId={props.hogFunctionId} />
-                        ),
-                    },
-                    ...columns.filter((column) => column.key !== 'logLevel'),
-                ]
+    const logicProps: LogsViewerLogicProps = {
+        sourceType: 'hog_function',
+        sourceId: props.hogFunctionId,
+    }
+
+    const { selectingMany, selectedForRetry, retryRunning } = useValues(hogFunctionLogsLogic(logicProps))
+    const { setSelectingMany, retrySelectedInvocations, selectAllForRetry } = useActions(
+        hogFunctionLogsLogic(logicProps)
+    )
 
-                return newColumns
-            }}
-        />
+    return (
+        <>
+            <PageHeader
+                buttons={
+                    <>
+                        {!selectingMany ? (
+                            <LemonButton size="small" type="secondary" onClick={() => setSelectingMany(true)}>
+                                Select invocations
+                            </LemonButton>
+                        ) : (
+                            <>
+                                <LemonButton size="small" type="secondary" onClick={() => setSelectingMany(false)}>
+                                    Cancel
+                                </LemonButton>
+                                <LemonButton size="small" type="secondary" onClick={() => selectAllForRetry()}>
+                                    Select all
```

Comment:
We should probably switch this into a `deselect all` if all events have been selected